canonical/canonical.com
# Fixed Naming for templates

## Done

- Removed "/" from names of templates in extended clause
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check if all the changed pages are working properly and resembles the original site
- Links to Demo Pages
   -  [Divesrity Page in Careers](https://canonical-com-1282.demos.haus/careers/company-culture/diversity)
   -  [Identity Page in Careers](https://canonical-com-1282.demos.haus/careers/company-culture/identity)
   - [Company Culture Careers](https://canonical-com-1282.demos.haus/careers/company-culture)
   - [Progression Page in Careers](https://canonical-com-1282.demos.haus/careers/company-culture/progression)
- Links to Original Pages at canonical.com
   -  [Diversity Page in Careers](https://canonical.com/careers/company-culture/diversity)
   -  [Identity Page in Careers](https://canonical.com/careers/company-culture/identity)
   - [Company Culture Careers](https://canonical.com/careers/company-culture)
   - [Progression Page in Careers](https://canonical.com/careers/company-culture/progression)
## Issue / Card

Fixes # [WD-12280](https://warthogs.atlassian.net/browse/WD-12280)

## Screenshots

[if relevant, include a screenshot]


[WD-12280]: https://warthogs.atlassian.net/browse/WD-12280?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
